GitHub-Test-4/wood-readingFileCollection.py

```diff
@@ -0,0 +1,114 @@
+# Our challenge: read in multiple text files from a directory:
+# Our resource: The Python os module + a handy code example:
+#  https://www.geeksforgeeks.org/how-to-read-multiple-text-files-from-folder-in-python/
+import spacy
+#nlp = spacy.cli.download("en_core_web_md")
+nlp = spacy.load('en_core_web_md')
+# AFTER THE FIRST DOWNLOAD, COMMENT OUT the spacy.cli.download(...) variable.
+# Your spaCy language model will already be stored in your Python environment.
+# ABOUT WHAT SPACY SHOULD LOAD: Some tutorials direct us to en_core_web_md
+# There are _sm, _md, and _lg models built into spaCy. Each takes up more space than the others, but
+# contains more data so may be more accurate/precise.
+# If we try the sm model, we're told that it does not have word vectors loaded, so it uses tagger, parser and NER (named
+# entity recognition to calculate similarity instead. Better to switch to the md model--but worth comparing results!
+import os
+# What is os? This is a library that allows Python to read your operating system (os).
+# This will let Python read and interpret file paths on your local computer.
+
+##############################
+# OBJECTIVE: Find out which words in my document are most similar to a particular word of interest
+# How to find this using spaCy similarity vectors?
+
+# Helpful resource for spaCy similarity calculation based on a selected word:
+# https://stackoverflow.com/questions/55921104/spacy-similarity-warning-evaluating-doc-similarity-based-on-empty-vectors
+##############################
+
+# ebb: The os module comes with python so you probably don't have to install it.
+# Just add the import line
+
+# commenting out in Pycharm: find keystroke under Code >> Comment with linecomment
+
+# ebb: Identify a file directory with text files to explore:
+# ebb: os.cwd returns the current working directory path
+
+workingDir = os.getcwd()
```

Comment:
@lindseyxwood Here you're getting the **current** working directory. `os.getcwd()`